Dockerfile

```diff
@@ -1,24 +1,15 @@
-FROM balena/open-balena-base:v11.0.2
+FROM balena/open-balena-base:no-systemd-master
```

Comment:
If this is not pinned to a specific tag because the PR is in Draft form, it's fine -- but it must definitely be pinned to a specific tag before merge.

Reply:
Seconded that this needs to stay using pinned versions

Reply:
+1

Reply:
@dfunckt @Page- this depends on https://github.com/balena-io/open-balena-base/pull/122 being merged, could you give this a review too?